alert-policies/temporal/ActivityEexcutionFail.yml

```diff
@@ -0,0 +1,31 @@
+# Name of the alert
+name: Activity Execution Fails
```

Comment:
@pkudikyala In filename did spelling mistake can you update it?

Reply:
Resolved review comments